pkg/objstore/s3store/main_test.go

```
@@ -18,18 +18,142 @@ import (
 	"context"
 	"fmt"
 	"io"
+	"net/http"
+	"path/filepath"
+	"strings"
 	"testing"
 
 	"github.com/aws/aws-sdk-go-v2/aws"
+	awsconfig "github.com/aws/aws-sdk-go-v2/config"
 	"github.com/aws/aws-sdk-go-v2/service/s3"
 	"github.com/pingcap/kvproto/pkg/brpb"
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(goimports)